recipes/pytest-chalice/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pytest-chalice" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1cf7ba41970d574315e5a2ae241476046b2fc674ce7d38d307560ddb4e433d3a
+  patches:
+    - setuptools_scm.patch
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  entry_points:
+    - chalice = pytest_chalice.plugin:main_entry_point
+  skip: True  # [win or py27]
```

Comment:
Is there any way that you can make this noarch python? You can use the `python >=35` in the requirements sections

Reply:
@scopatz works as noarch - added it in with the python pinnings.